components/quest-boost/boostCard.tsx

```diff
@@ -97,31 +105,46 @@ const BoostCard: FunctionComponent<BoostCardProps> = ({
           ) : (
             <div className="flex w-full">
               <div className="flex items-center">
-                <div className={styles.issuer}>
-                  {boost.expiry > Date.now() ? (
-                    hasUserCompletedBoost ? (
-                      <>
-                        <p className="text-white">Done</p>
-                        <CheckIcon width="24" color="#6AFFAF" />
-                      </>
-                    ) : null
-                  ) : boost.winner === null ? (
-                    <>
-                      <p className="text-white">Done</p>
-                      <CheckIcon width="24" color="#6AFFAF" />
-                    </>
-                  ) : isClickable ? (
-                    <>
-                      <p className="text-white">See my reward</p>
-                      <TrophyIcon width="24" color="#8BEED9" />
-                    </>
-                  ) : (
-                    <>
-                      <UnavailableIcon width="24" color="#D32F2F" />
-                      <p className="text-white mr-2">Boost ended</p>
-                    </>
-                  )}
-                </div>
+                {address && (
+                  <div className={styles.issuer}>
+                    {boost.expiry < Date.now() ? (
+                      hasUserCompletedBoost ? (
+                        !userBoostCheckStatus ? (
+                          boost.winner != null ? (
+                            <>
+                              <p className="text-white">See my rewards</p>
+                              <CheckIcon width="24" color="#6AFFAF" />
```

Comment:
i think there is a trophy icon here for see my rewards and also can you update the token addresses for the boost in so that the amount tag is rendered for all. I think currently only strk token renders
<img width="1476" alt="Screenshot 2024-04-14 at 2 02 43 AM" src="https://github.com/starknet-id/starknet.quest/assets/41674634/4db9c19f-3e2c-4493-97b8-51c08c7d585a">